components/org.wso2.carbon.consent.mgt.core/src/main/java/org/wso2/carbon/consent/mgt/core/model/PIICategoryValidity.java

```diff
@@ -36,6 +36,13 @@ public PIICategoryValidity(Integer id, String validity) {
         this.validity = validity;
     }
 
+    public PIICategoryValidity(Integer id, String validity, boolean isConsented) {
```

Comment:
any reason for not using primitive type here for id?

Reply:
Fixed https://github.com/wso2/carbon-consent-management/pull/184/commits/b806239544def31f36687998e3f3570b9c81d4d2